deploy/auxiliary/caf-secure-hubSpoke.json

```
@@ -0,0 +1,244 @@
+                "allowForwardedTraffic": true,
+                "allowGatewayTransit": true,
+                "useRemoteGateways": false
+            }
+        },
+        // Virtual network peering hub to spoke one
+        {
+            "type": "Microsoft.Network/virtualNetworks/virtualNetworkPeerings",
+            "apiVersion": "2020-05-01",
+            "name": "[concat(variables('hubNetwork').name, '/peer-to-', variables('spokeNetworkOne').name)]",
+            "location": "[variables('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('hubNetwork').name)]",
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('spokeNetworkOne').name)]"
+            ],
+            "properties": {
+                "allowVirtualNetworkAccess": true,
+                "allowForwardedTraffic": true,
+                "allowGatewayTransit": true,
+                "useRemoteGateways": false,
+                "remoteVirtualNetwork": {
+                    "id": "[resourceId('Microsoft.Network/virtualNetworks', variables('spokeNetworkOne').name)]"
+                }
+            }
+        },
+        // Virtual network peering hub to spoke two
+        {
+            "type": "Microsoft.Network/virtualNetworks/virtualNetworkPeerings",
+            "apiVersion": "2020-05-01",
+            "name": "[concat(variables('hubNetwork').name, '/peer-to-', variables('spokeNetworkTwo').name)]",
+            "location": "[variables('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('hubNetwork').name)]",
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('spokeNetworkTwo').name)]"
+            ],
+            "properties": {
+                "remoteVirtualNetwork": {
+                    "id": "[resourceId('Microsoft.Network/virtualNetworks', variables('spokeNetworkTwo').name)]"
+                },
+                "allowVirtualNetworkAccess": true,
+                "allowForwardedTraffic": true,
+                "allowGatewayTransit": true,
+                "useRemoteGateways": false
+            }
+        },
+        // Virtual network peering spoke two to hub
+        {
+            "type": "Microsoft.Network/virtualNetworks/virtualNetworkPeerings",
+            "apiVersion": "2020-05-01",
+            "name": "[concat(variables('spokeNetworkTwo').name, '/peer-to-', variables('hubNetwork').name)]",
+            "location": "[variables('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('hubNetwork').name)]",
+                "[resourceId('Microsoft.Network/virtualNetworks', variables('spokeNetworkTwo').name)]"
+            ],
+            "properties": {
+                "allowVirtualNetworkAccess": true,
+                "allowForwardedTraffic": true,
+                "allowGatewayTransit": true,
```

```suggestion
                "allowGatewayTransit": false,
```
Typically only needed in hub to spoke peering, and only to support on-prem VPN Gateway/ER.

---

Done